src/Tribe/Plugins_API.php

```diff
@@ -288,10 +288,15 @@ public function get_products() {
 				'link'             => 'https://evnt.is/1bd7',
 				'plugin-dir'       => 'event-tickets-wallet-plus',
 				'main-file'        => 'event-tickets-wallet-plus.php',
-				'description'      => __( 'Allow event attendees to save their tickets to digital wallets, like Apple Wallet or PassWallet, and download PDF tickets, to have them with them at any time.', 'tribe-common' ),
+				'description'      => __( 'Make it simple for your attendees to save and find their tickets.', 'tribe-common' ),
 				'description-help' => __( 'Allow event attendees to save their tickets to digital wallets.', 'tribe-common' ),
-				'features'         => [],
-				'image'            => 'images/shop/wallet-plus.svg',
+				'features'         => [
+					__( 'Flexible ticket formats', 'tribe-common' ),
+					__( 'Easy day-of-event management', 'tribe-common' ),
+					__( 'Email and checkout integration', 'tribe-common' ),
+					__( 'Ticket portability', 'tribe-common' ),
+				],
+				'image'            => 'images/shop/wallet-plus.jpg',
 				'logo'             => 'images/logo/wallet-plus.svg',
```

Comment:
We're keeping the svg here, but I see you're removing the svg file as part of the PR. Is this affecting anything in some capacity? Maybe the help page?

Reply:
Oh yeah, I mixed the path. Awsome then, thanks for the clarification 🥇